glide.yaml

```diff
@@ -1,8 +1,7 @@
 package: go.uber.org/fx
 import:
-- package: github.com/uber-go/zap
-  # Pinned temporarily while zap reaches 1.0 (https://github.com/uber-go/fx/pull/243)
-  version: c064b5c44b285a7e2fd5c9b26e8c38228ce2bccb
+- package: go.uber.org/zap
+  version: v1.0.0-rc.1
```

Comment:
Cool. Again though, this makes me think we should keep a ulog.Logger even if it's currently just the exact same interface as a zap.Logger so that we can upgrade to v1 without breaking users.